test/navigation/NavigatorSpec.scala

```diff
@@ -582,7 +592,7 @@ class NavigatorSpec extends SpecBase {
               BusinessContactDetailsPage,
               NormalMode,
               ua
-            ) mustBe routes.ValuationMethodController.onPageLoad(NormalMode, draftId)
+            ) mustBe routes.ChoosingMethodController.onPageLoad(draftId)
```

Comment:
Change method name to: "navigate to choose valuation method page when Yes" (like below).